tasks/main.yml

```diff
@@ -1,36 +1,41 @@
-- name: Set database to mysql
-  debconf:
-    name: sympa
-    question: sympa/database-type
-    value: mysql
-    vtype: select
+---
+## Sympa configuration main tasks file
 
-- name: Set webserver to other
-  debconf:
-    name: sympa
-    question: wwsympa/webserver_type
-    value: Other
-    vtype: select
+- name: Check that "debconf" and "debconf-utils" are installed
+  apt:
+    name:
+    - debconf
+    - debconf-utils
+    state: latest
```

Comment:
I added this requirements' check, as [suggested by Ansible documentation](https://docs.ansible.com/ansible/latest/modules/debconf_module.html#requirements), even if it seems to work without `debconf-utils` package.